### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by proconsule** [mpv-player/mpv#9464]
Is there any chance to have audio sample float (in some way) accessible in the user shader code, this will help making cool shaders for audio playback (like old winamp vis plugins).

I know this is hard (buffer, sync and other problems) but will open great sceanrios on making players based on mpv

**Comment by thebombzen** [comment]
Shaders run on a GPU, are you perhaps referring to audio filters? You can use that with `--af=`.

**Comment by CounterPillow** [comment]
no, he is referring to wanting to access audio sample data in a gpu shader to do visualisations

**Comment by kasper93** [comment]
I think it would be not so trivial to support. Patches welcome if someone wants to deep dive into it.
